src/samples/Authorization/AutoSignIn/Program.cs

```diff
@@ -22,8 +23,8 @@
 // Add AspNet token validation
 builder.Services.AddAgentAspNetAuthentication(builder.Configuration);
```

Comment:
This isn't require since the AspNet authentication isn't enabled.

Reply:
I was going to exclude them from the version that I push to agent repo samples.

Reply:
If we dont add AspNet authentication, this sample won't be ready to be deployed to a real world.

I think all samples should enable aspnet auth.

Reply:
I think this is tied to Https, If I run Http connections it seems to work fine.. if I run https connections it fails w/out this file.. its interesting and quriky.

Reply:
For now going to leave it in this repo